LanguageFeatures/Patterns/map_A04_t01.dart

```diff
@@ -52,8 +59,16 @@ String test1(Map map) {
 //              ^
 // [analyzer] unspecified
 // [cfe] unspecified
-    case {c1: var a, c2: final b} => "";
-//                   ^^
+    case {c1: var a1, c2: final b1} => "";
+//                    ^^
+// [analyzer] unspecified
+// [cfe] unspecified
+    case {3.14: var a2, 3.14: final b2} => "";
```

Comment:
Yes! With the new rule about errors for identical keys, `double` keys _are_ covered. Nice! ;-)